src/scribe_data/cli/list.py

```diff
@@ -0,0 +1,153 @@
+import json
+from pathlib import Path
+from typing import Dict, List, Union
+
+# Load language metadata from JSON file
```

Comment:
Inline comments should have a period at the end of them as they're complete sentences :)